docs/modules/plugins/pages/plugin-web-app-to-rest-api.adoc

```diff
@@ -182,6 +182,16 @@ a|`true`
 |`true`
 |Whether to crawl https pages.
 
+|`transformer.from-headless-crawling.default-headers.<header name>=<header value>`
```

Comment:
```suggestion
|`transformer.from-headless-crawling.http.header.<header name>=<header value>`
```

Reply:
but we override default class, and thus can keep consistent mapping in properties

Reply:
as to consistency, 90% properties mentioned here https://docs.vividus.dev/vividus/latest/plugins/plugin-web-app-to-rest-api.html#_from_headless_crawling are about http, but we didn't prefix them with http

Reply:
https://github.com/vividus-framework/vividus/blob/4618e8aadc833c49b7e2f3d70b3d7ff8eb2519ca/vividus-plugin-rest-api/src/main/resources/vividus-plugin/spring.xml#L58

https://github.com/vividus-framework/vividus/blob/4618e8aadc833c49b7e2f3d70b3d7ff8eb2519ca/vividus-plugin-web-app-to-rest-api/src/main/resources/vividus-plugin/spring.xml#L59